### ISSUE TITLE
Annotation Problems with release 2.0.1

### ISSUE THREAD

**Issue by pierre-michalet** [bvaughn/angular-form-for#102]
Hello,

I tried the 2.0.1 version but it breaks with strictdi enabled. It seems that the js is no longer correctly annotated.

I tried to annotate it with the last release of ng-annotate but it didnt' work.

Has anyone experienced the same problem ?

Thanks

**Comment by bvaughn** [comment]
Hmm. You're the first one to report it, but I'm not sure how many people have upgrade to the 2.x or 3.x builds yet. They're still new.

At a glance, I'm not sure what's wrong. [The built dist](https://github.com/bvaughn/angular-form-for/blob/master/dist/form-for.js) seems to have the correct bracket syntax.

Any ideas?

**Comment by pierre-michalet** [comment]
We have occurrences of
```javascript
controller: function($scope) ...
```

and it should be 
```javascript
controller: ["$scope", function($scope) ...
```
like we had in 1.x.x

I'm not sure why ng-annotate doesn't work in this case. Maybe adding @ngInject comments can help

**Comment by bvaughn** [comment]
I've got a handle on this. Seems like I need to give `ngAnnotate` a bit of a nudge for some of my TypeScript classes, by way of an `/* @ngInject */` comment. I'll try to get a release built and pushed out this evening for you.

By chance, do you need 2.x to be updated? Or would you be willing to upgrade to 3.x? (3.x is backwards compatible except for [the change to radio fields](https://github.com/bvaughn/angular-form-for/blob/master/CHANGELOG.md).)

**Comment by pierre-michalet** [comment]
3.x should be good for me

Thanks for your reactivity !

**Comment by bvaughn** [comment]
Fantastic. :)

I'm still looking into it. Thought I'd fixed it but then some of my regressions failed and it had gotten late. I'll try to get a release out tonight or tomorrow. :)

**Comment by bvaughn** [comment]
I'm pleased to inform you that release 3.0.1 is out and should resolve your strict-di issues. :)

Thanks for reporting this. Please let me know if you encounter any other issues while upgrading.

**Comment by bvaughn** [comment]
Re-opening this issue because I may have noticed a problem with field-label, post release. Will report back with more soon.

**Comment by bvaughn** [comment]
Ok. My apologies for that. Resolved a minor fieldLabel issue with 3.0.2. You should now be good to go. I've updated the documentation site to also run with strict-di so that I'll notice this before releasing next time. :)
